src/os/bluestore/BlueStore.cc

```diff
@@ -5513,7 +5553,7 @@ int BlueStore::_open_db_and_around(bool read_only)
       goto out_db;
     }
 
-    r = _open_fm(nullptr);
+    r = _open_fm(nullptr, false);
```

Comment:
I guess there is an inversion between wording ("db_in_read_only") and actual function.